src/composables/graph/useSelectionState.ts

```diff
@@ -94,6 +94,19 @@ export function useSelectionState() {
     computeSelectionStatesFromNodes(selectedNodes.value)
   )
 
+  // Keep help panel in sync when it is open and the user changes selection.
+  watch(
+    () => nodeDef.value,
```

Comment:
Optional: Why a getter here if you're checking the value of the ref itself?

```suggestion
  nodeDef
```

Reply:
good points, changes made